README.md

```diff
@@ -82,12 +82,12 @@ This is the main Component, build around a `ScrollView` component, that should w
 |containerStyle| [View Style](https://reactnative.dev/docs/view-style-props) | Container Styles passed to the `ScrollView` |
 |keyboardHiddenContainerStyle| [View Style](https://reactnative.dev/docs/view-style-props) | Container Styles passed to the `ScrollView`. Applied only when the keyboard is hidden |
 |scrollViewProps| `Object` - [ScrollView Props](https://reactnative.dev/docs/scrollview#props) | Props passed to the `ScrollView`|
-|safeMarginContentHeight| `Number` | Safe Margin to determine if content fits between top of screen and keyboard. |
-|safeMarginBottom| `Number` | Safe Margin between slected input or form and keyboard |
+|safeMarginContentHeight| `Number` | Safe Margin to determine if content fits between top of screen and keyboard. `contentHeight` is the height of the elements that are withing the `<Avoidable>` or `<Avoidable.Area>` tags. You might want to fine-tune this so by changing the `safeMarginContentHeight` you can add or subtract from it. Default `0` |
```

Comment:
I don't really understand what this "Safe Margin" is. On line 86, you explain very clearly what that is. Here I'm a bit confused. 
I would also remove the sentence that start with "You might want to fine-tune...". What do you think?